entity-framework/core/miscellaneous/multitenancy.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Multi-tenancy - EF Core
+description: Learn several ways to implement multi-tenant databases using Entity Framework Core.
+author: jeremylikness
+ms.author: jeliknes
+ms.date: 12/17/2021
+uid: core/miscellaneous/multitenancy
+---
+# Multi-tenancy
+
+Many line of business applications are designed to work with multiple customers. It is important to secure the data so that customer data isn't "leaked" or seen by other customers and potential competitors. These applications are classified as "multi-tenant" because each customer is considered a tenant of the application with their own set of data.
+
+> [!IMPORTANT]
+> This document provides examples and solutions "as is." These are not intended to be "best practices" but rather "working practices" for your consideration.
+
+## Supporting multi-tenancy
+
+There are many approaches to implementing multi-tenancy in applications. One common approach (that is sometimes a requirement) is to keep data for each customer in a separate database. The schema is the same but the data is customer-specific. Another approach is to partition the data in an existing database by customer.
```

Comment:
When partitioning multiple tenants in the same database, there's the single-schema approach with a tenant ID (which is where global query filters come into play), but there's also the schema-per-tenant approach, where schemas can also diverge between tenants (big headache). It may be worth at least making that distinction and clarify which one we're talking about here, etc.

Ultimately we may want multiple nested pages showing each method: multiple databases, multiple schemas, single schema with tenant ID; it may be slightly confusing to have the two approaches described below in the same page. But of course we don't need to cover everything right away (as it's a bit subject), I'd be happy to contribute to this as well in the future.

Reply:
Great feedback. I added an example.